### DISCUSSION TITLE
automatic import of custom code

### DISCUSSION THREAD

**Discussion by farhad-sh** [joomengine/Joomla-Component-Builder#691] [Extention Issues]
Hello @Llewellynvdm 
In some parts of my project I have to use custom codes (Automatic import of custom code during compilation) as you do in this video: https://www.youtube.com/watch?v=DFMfIl-VkSk&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=32
The only difference is that I put the /***[REPLACE<>$$$$]***/ and /***[/REPLACE<>$$$$]***/ tags in the admin/components /com_myComponent /views/myViews/tmpl/default_body.php , but after compiling and installing the /***[REPLACED$$$$]***//*x*/ and /***[/REPLACED$$$$]***/ tags do not fit in the location I specified.
i'm doing this because of changes to the default_body.php file . is that correct? or can automatic custom code be used anywhere?

**Comment by Llewellynvdm** [comment]
This topic is either very simple or very complicated... I tried to explain in a number of tutorials.. of which you mentioned one. Not sure if you watched them all, but here is the list:
https://www.youtube.com/watch?v=H3M1NRaNOzg&list=PLQRGFI8XZ_wuzAQELTUUIDdqKFmLP9lSl&index=9
https://www.youtube.com/watch?v=DFMfIl-VkSk&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=31
https://www.youtube.com/watch?v=KiAtJawZ3oo&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=32
https://www.youtube.com/watch?v=8Yl4lAAAWMQ&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=71
https://www.youtube.com/watch?v=FEJ8WEbcTys&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=87
https://www.youtube.com/watch?v=QGMURJI_Mds&list=PLQRGFI8XZ_wuzAQELTUUIDdqKFmLP9lSl&index=10

Make sure to watch all these... and let me know if you have any further questions :+1:

**Reply by farhad-sh** [reply]
i saw the videos and my problem was solved .
thank you for your guidance❤👌
